testing/ssh-honeypot/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="ssh-honeypot"
+pkgver="0.1.0"
+_commit=b7baa9a6963e723ee0779a6cef0447739870ace3
+pkgrel=0
+pkgdesc="Fake sshd that logs ip addresses, usernames, and passwords."
+url="https://github.com/droberson/ssh-honeypot"
+arch="all"
+license="MIT"
+depends="openssh-keygen"
+makedepends="clang json-c0.12-dev libssh-dev"
+options="!check"
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.zip::https://github.com/droberson/ssh-honeypot/archive/$_commit.zip
+	$pkgname.initd
+	$pkgname.confd"
+builddir="$srcdir/$pkgname-$_commit"
+
+
+package() {
+	mkdir -p $pkgdir/usr/bin/ $pkgdir/usr/share/ssh-honeypot
+	make PREFIX=/usr DESTDIR="$pkgdir" -C "$builddir"
+	install -m 755 -D bin/$pkgname $pkgdir/usr/bin/$pkgname
+	install -m 755 -D "$srcdir"/$pkgname.initd $pkgdir/etc/init.d/$pkgname
+	install -m 644 -D "$srcdir"/$pkgname.confd $pkgdir/etc/conf.d/$pkgname
```

Comment:
quote $pkgname too (it's possible that it contains a space)

Reply:
I think you meant $pkgdir ;)

Reply:
Uhh...right :P